pretext/project/__init__.py

```diff
@@ -1326,6 +1326,15 @@ def _get_target(
             # return first target matching the provided name
             return next(t for t in self.targets if t.name == name)
         except StopIteration:
+            # if there is no target matching, check if there is a format
+            # that matches. Use that as a backup with a warning
+            for target in self.targets:
+                if target.format == name:
```

Comment:
I have a slight preference to ensure `name` is a valid `format` here, but it's not a strong preference.

Reply:
Shouldn't `format` be a valid format unless the person's `project.ptx` is not schema-compliant?

Reply:
yeah, I think that is sufficient guardrails now that you mention it.